src/EFCore.PG/Storage/Internal/Mapping/NpgsqlListTypeMapping.cs

```diff
@@ -51,25 +55,27 @@ public NpgsqlListTypeMapping(RelationalTypeMapping elementMapping, Type listType
 
         /// <inheritdoc />
         NpgsqlListTypeMapping(string storeType, RelationalTypeMapping elementMapping, Type listType)
-            : this(new RelationalTypeMappingParameters(
-                new CoreTypeMappingParameters(listType, null, CreateComparer(elementMapping, listType)), storeType
-            ), elementMapping) {}
+            : this(
+                new RelationalTypeMappingParameters(
+                    new CoreTypeMappingParameters(listType, null, CreateComparer(elementMapping, listType)), storeType
+                ),
```

Comment:
Save the line, move it one line up (:

Reply:
Thanks, was specifically trying to catch these, but missed this one.